CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 ## Next release
 
+## 0.38.0 / 2020-03-20
+
 * [CHANGE] Changed ThanosRuler custom resource field alertmanagersURL type from string to []string` (#3067)
+* [CHANGE] Deprecate PodMonitor targetPort field (#3071)
```

Comment:
This links to a closed issuee 🤔 
https://github.com/coreos/prometheus-operator/issues/3071

Reply:
Did you mean https://github.com/coreos/prometheus-operator/pull/3078?
Note discussion here, should that have been merged 🤔 https://github.com/coreos/prometheus-operator/issues/3089
cc @brancz

Reply:
#3071 was intentional since that is the reported issue that prompted the change.  I was thinking about whether we should put the issue number or pr number or both in the changelog entries.  I think it makes sense to put the issue number since that usually describes the reason for the change from the user's perspective, but maybe I we should also include the PR number?

The change doesn't break anything in the API, it just discourages use of the field by marking it deprecated in the docs and logging a warning when it's used.  So I think it's safe to include in the release.

Reply:
Sorry my mistake, thought we removed it.

Makes sense but can we add both link to issue and PR the change was made in to the changelog line.

Reply:
Done!